packages/wonder-blocks-spacing/components/strut.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+/**
+ * A component for inserting fixed space between components.
+ *
+ * Assumes parent is a View.
+ */
+
+import * as React from "react";
+
+type Props = {
+    size: number,
+};
+
+export default class Strut extends React.Component<Props> {
+    render() {
+        return <div style={flexBasis(this.props.size)} />;
```

Comment:
🤷‍♂️ Do we want to make it possible to *only* specify the spaces that we having in `spacing`?

🤷‍♂️ it's be useful if we could do: `size="xSmall"` or similar!

Reply:
We could give that a try.